src/components/experimental/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode, type ReactElement } from 'react';
+import styled from 'styled-components';
+import { SpaceProps, LayoutProps, PositionProps, FlexboxProps } from 'styled-system';
+
+import { get } from '../../../utils/experimental/themeGet';
+import { getSemanticValue } from '../../../essentials/experimental';
+import { textStyles } from '../Text/Text';
+import { XCrossIcon } from '../../../icons';
+
+const Container = styled.div`
+    position: relative;
+    justify-content: space-between;
+
+    border: none;
+    outline: none;
+    border-radius: ${get('radii.4')};
+    padding: ${get('space.3')} ${get('space.4')};
+    color: ${getSemanticValue('inverse-on-surface')};
+    background-color: ${getSemanticValue('inverse-surface')};
+
+    display: inline-flex;
+    align-items: center;
+    gap: ${get('space.1')};
+
+    ${textStyles.variants.body2}
+`;
+
+interface SnackbarProps extends SpaceProps, LayoutProps, PositionProps, FlexboxProps {
+    children: ReactNode;
+    hasDismissButton?: boolean;
+    onDismiss?: () => void;
+}
+
+const Snackbar = ({
+    children,
+    hasDismissButton = false,
+    onDismiss = null,
+    ...restProps
+}: SnackbarProps): ReactElement => (
+    <Container {...restProps}>
+        {children}
+        {hasDismissButton && (
+            <XCrossIcon
+                data-testid="snackbar-close-icon"
```

Comment:
True 😅 I guess it was passing the tests due to the svg's onclick attribute.
I used the `IconButton` component, unsetting width and height in order to prevent affecting the overall dimensions of the Snackbar. Let me know what you think.
Thanks!